lib/seahorn/BvOpsem2WideMemMgr.cc

```diff
@@ -542,13 +538,29 @@ class WideMemManager : public OpSemMemManagerBase {
     LOG("opsem", WARN << "setFatData() not implemented!\n");
     return nullPtr();
   }
+
+  Expr isDereferenceable(PtrTy p, Expr byteSz) {
```

Comment:
add also a version with the signature `Expr isDereferenceable(PtrTy p, unsigned bytes)` and re-direct to it. We might be able to support numeric ranges differently than symbolic.